bertopic/representation/_openai.py

```diff
@@ -37,7 +38,7 @@
 Topic name:"""
 
 DEFAULT_CHAT_PROMPT = """
-I have a topic that contains the following documents: 
+I have a topic that contains the following documents:
```

Comment:
Although I expect no issues, it would be interesting to see the effect of removing a `\n` from this prompt. In my experience with recent LLMs (Phi-3/Llama-3), this actually can have a decent influence on performance (especially with respect to following instructions). Either, good change!

Reply:
That's a great point. I am tempted to undo this change then I don't necessarily want to modify any behavior other than logging.

Reply:
Let's keep it for now and potentially change this in the future.